src/webid-oidc.js

```diff
@@ -136,6 +139,23 @@ const sendAuthRequest = async (
   return navigateTo(url)
 }
 
+async function saveAppHashFragment(store): Promise<void> {
+  await updateStorage(store, data => ({
+    ...data,
+    appHashFragment: window.location.hash
+  }))
+}
+
+async function restoreAppHashFragment(store): Promise<void> {
+  let appHashFragment
+  await updateStorage(store, data => {
+    appHashFragment = data.appHashFragment
+    delete data.appHashFragment
+    return data
+  })
+  window.location.hash = appHashFragment
```

Comment:
(nitpick): wondering why we don't eliminate the variable `appHashFragment` by just setting `.hash` directly.

Reply:
yes, this does look odd on second-pass :) I'll make the change.